### ISSUE TITLE
Pre-process crates method on Pallet

### ISSUE THREAD

**Issue by stdavis** [agrc/forklift#190]
### Use Case
The OGM pallet updates data in SGID. I would like this data updated prior to any crates being processed so that the crates get the updates in SGID and they don't have to wait for the next day.

Do we need to add a new method to `Pallet` that is called before `process_crates_for`? The update logic for the OGM pallet used to be in `build` but that's not going to work now that it's called on every lift.

**Comment by steveoh** [comment]
Would this method have any logic behind having it called or would it be another build method with a new name?

**Comment by stdavis** [comment]
I don't think that it would be any different from the old implementation of the build method (only called for pallets that are lifted and error trapped).

**Comment by steveoh** [comment]
should we call it `prepare_packaging`, `drop_ship`, `stock`, `pinwheel`, `sideshift`, `stamp`

[This resource](http://www.inventoryops.com/dictionary.htm) has all of the definitions and some some good names for things :)

**Comment by stdavis** [comment]
I like `prepare_packaging`.

**Comment by steveoh** [comment]
you have my blessing
